openapi/assessment_config.yml

```diff
@@ -0,0 +1,79 @@
+components:
+  schemas:
+    Answer:
+      type: object
+      properties:
+        field_id:
+          oneOf:
+            - type: string
+            - type: array
+              items:
+                type: string
+        form_name:
+          type: string
+        field_type:
+          type: string
+        presentation_type:
+          type: string
+        question:
+          type: string
+    Theme:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        answers:
+          type: array
+          items:
+            $ref: '#/components/schemas/Answer'
+    SubCriteria:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        themes:
+          type: array
+          items:
+            $ref: '#/components/schemas/Theme'
+    UnscoredCriteria:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        sub_criteria:
+          type: array
+          items:
+            $ref: '#/components/schemas/SubCriteria'
+    ScoredCriteria:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        weighting:
+          type: number
+          format: float
+        sub_criteria:
+          type: array
+          items:
+            $ref: '#/components/schemas/SubCriteria'
```

Comment:
⛏️ No need for both of these, if you pick the bigger schema you can just use that. The most important thing is that `scored_criteria` **requires** a weighting. See https://github.com/communitiesuk/funding-service-design-notification/blob/main/openapi/components.yml#L147 for how to inherit from a schema and make a property required.

Reply:
Thanks! Added here: https://github.com/communitiesuk/funding-service-design-assessment-store/commit/ff4aa11dbf05cb6f41bf0779f4efccc36d5a3c6d#diff-ae0012ebd0e74874b167156eff37d9e47b8f7a7ec78ba7fb499078a7537c371d